Files/UserControls/NavigationToolbar.xaml.cs

```diff
@@ -1219,15 +1219,15 @@ private void PathboxItemFlyout_Opened(object sender, object e)
 
         private void VerticalTabStripInvokeButton_Loaded(object sender, RoutedEventArgs e)
         {
-            if (!(MainPageViewModel.MultitaskingControl is VerticalTabViewControl))
+            if (!(mainPage.ViewModel.MultitaskingControl is VerticalTabViewControl))
```

Comment:
Eventually, we won't need to rely on the Window.Current hack once VerticalTabViewControl is moved to be outside of NavigationToolbar